### ISSUE TITLE
Adding markdown to FCC curriculum

### ISSUE THREAD

**Issue by thecodingaviator** [freeCodeCamp/freeCodeCamp#34140]
I was wondering that since markdown is a part of web development/design and people writing blogs or guide articles such as ours would surely at some time encounter markdown which for me, wasn't friendly for the first week or so, so maybe we could have a markdown challenges section added?

**Comment by Nirajn2311** [comment]
@thecodingaviator Along with markdown we could also add some challenges for teaching **git**.

**Comment by thecodingaviator** [comment]
@Nirajn2311 Yup, we should

**Comment by erictleung** [comment]
@Nirajn2311 thanks for the suggestion. This has been brought up before https://github.com/freeCodeCamp/freeCodeCamp/issues/12479. But like all things, there were issues of higher priority, such as getting the now current site up and running.

From what I gathered from that discussion, 

- we don't want to add more to the Getting Started bit,
- we want people learning Markdown to be interactive i.e. not just a GIF of Markdown syntax
- have the intro be close to where Gitter is introduced

Here are some resources people suggested:

- [Gitter's Markdown Basics](https://gitter.zendesk.com/hc/en-us/articles/200176682-Markdown-basics) - quick and simple run down of Markdown
- [How to Post Code in Gitter Chat for Free Code Camp](https://youtu.be/ITu5kyiVtDw) (2:33) - nice visual and commentary demonstrating how to post in Markdown
- [10 minute Interactive Markdown Tutorial](https://commonmark.org/help/) - very clean interactive tutorial, but does take 10 minutes.

All three have their strenghts and weaknesses. 

And [here's](https://github.com/freeCodeCamp/freeCodeCamp/issues/14348) another very similar issue where it could be nice to have a quick "Copy" button where it autoformats your code for Markdown.

Knowing Markdown will also help in formatting code in the forums, where I've seen many (very patient) admins there helping format people's code.

**Comment by raisedadead** [comment]
Hi @thecodingaviator I am inclined to the idea. And maybe this could go into one of our non-certification offerings, definitely not in the getting started section.

Although, we would like to overhaul the some of the other pressing areas of the curriculum. Going off-topic here for a bit, we could love a hand at updating the Bootstrap curriculum (currently still using BootStrap 3) for example.

P.S: There is a thread about Git https://github.com/freeCodeCamp/freeCodeCamp/issues/16390 which may be taken prior to this, for instance.

**Comment by Nirajn2311** [comment]
@thecodingaviator @raisedadead this feature was discussed before in the curriculum repo - 
https://github.com/freeCodeCamp/curriculum/issues/273

**Comment by thecodingaviator** [comment]
@erictleung Yup, I definitely agree about not adding it to get started. And @raisedadead I think for maintenance of old technology, a new issue can be opened, I'm still new to the FCC OSS side and am not sure how things work around here. :)

**Comment by thecodingaviator** [comment]
@erictleung / @QuincyLarson Any updates / decisions on this?

**Comment by HugoLiconV** [comment]
@raisedadead I would like to help to updating the Bootstrap curriculum. Is there an issue open for that?

**Comment by thecodingaviator** [comment]
I think we can add it in the APIs and Microservices certification, somewhere next to the npm challenges.
